compiler/rustc_codegen_ssa/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 test = false
 
 [dependencies]
-ar_archive_writer = "0.1.1"
+ar_archive_writer = "0.1.2"
```

Comment:
I put up https://github.com/rust-lang/rust/pull/109562 a couple of days ago for a bump to 0.1.3.